tests/unit/contracts/test_layers.py

```diff
@@ -30,6 +30,17 @@ def configure():
                 independent=True,
             ),
         ),
+        (
+            "one : two :    three",
+            Level({Layer(name="one"), Layer(name="two"), Layer(name="three")}, independent=False),
+        ),
+        (
+            "one : (two) : three",
```

Comment:
Could we have a test for the invalid mixture of two kinds of dividers?

Reply:
Yes good idea, added